dist/backdraft.js

```diff
@@ -575,8 +581,101 @@ _.extend(Plugin.factory, {
         return columnType;
       }
     });
+  },
+
+  _applySavedState: function() {
+    var reportSettings = this.table.options.reportSettings;
```

Comment:
While I understand that the user settings are being stored in the Invoca ReportSettings model/table, these changes are being made to the Backdraft public repo. Can we rename references in Backdraft to something more generic? Like "dataTableSettings?" or "userPreferences?" or something like that?